react/week2/todo-app/src/AddTodo.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import addTodo from "./App";
+import todoItems from "./todoItems.json";
+
+function AddTodo({ addTodo }) {
+	const [value, setValue] = useState("");
+	const [deadline, setDeadline] = useState("");
+	const [date, setDate] = useState("");
+
+	return (
+		<div className="add-todo-form">
+			<form onSubmit="">
+				<label>
```

Comment:
Please try to use a beautify extension for your code editor, so you get a clean code,
this code is from week2 please always check your code and when you push something to your week3 branch avoid pushing changed code from week2 thanks.